mne_connectivity/spectral/tests/test_spectral.py

```diff
@@ -472,7 +472,7 @@ def test_epochs_tmin_tmax(kind):
     assert len(w) == 1  # just one even though there were multiple epochs
 
 
-@pytest.mark.parametrize('method', ['coh', 'plv', 'pli', 'wpli'])
+@pytest.mark.parametrize('method', ['coh', 'plv', 'pli', 'wpli', 'ciplv'])
```

Comment:
Do you mind adding a separate test for these where `freqs` is set and asserting some behavior? I.e. that certain frequency bins are used vs the default done in `multitaper`?

Perhaps a separate test for `test_spectral_connectivity_time_multitaper`?

Reply:
I agree that a test is necessary, will do.

Reply:
I realized this is a bit troublesome. Namely, we currently require neither `freqs` nor `fmin`/`fmax`. This leads to the issue of setting sensible defaults for `fmin` and `fmax`. 

Essentially, `fmin` should be set such that the wavelet with the set `n_cycles` doesn't become longer than the signal. However, even with the length of the wavelet equal to that of the signal, the spectral estimate isn't very good and thus we cannot `assert connectivity == 1` for synchronized data.

There's still another problem with `freqs` and `n_cycles` which I noticed while looking into this. Essentially, `n_cycles` should either be a single number or the same length as `freqs`. This could lead to confusion, as the length of 'freqs' is not known by the user unless its set by them. One option would be removing '_compute_freqs' and just requiring 'freqs' like it used to be before I started tampering with the code.

Reply:
Sounds good to me. I think we're allowed to make breaking changes as long as the results work rn. The package is not stable by any means.